src/sdk/namespace_fs.js

```diff
@@ -1257,6 +1257,9 @@ class NamespaceFS {
         const part_upload = file_path === upload_path;
         const same_inode = params.copy_source && copy_res === copy_status_enum.SAME_INODE;
         const is_dir_content = this._is_directory_content(file_path, params.key);
+        // upload_part should disable ctime_check because we update the same part-file on concurrent put part 
+        const disable_ctime_check = part_upload;
+        fs_context = { ...fs_context, disable_ctime_check };
```

Comment:
Do we need this for all the operations inside finish_upload? couldn't it have unexpected outcomes - for example if we use safe_rename_posix?

Can we be selective and make an explicit fs_context to be used only for those stat calls in question -
```suggestion
        const fs_context_for_part_stat = { ...fs_context, disable_ctime_check: part_upload };
```

Reply:
On finish_upload + upload_part=true we are not doing safe_rename_posix or almost anything actually 
but sure, i'll add it to stats only in order to not cause unwanted behavior